KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift

```diff
@@ -149,6 +153,45 @@ final class KarhooBookingViewController: UIViewController, BookingView {
         navigationItem.backButtonTitle = ""
     }
 
+    private func setupBottomContainer() {
+        // bottom container
+        bottomContainer = UIView()
+        bottomContainer.translatesAutoresizingMaskIntoConstraints = false
+        bottomContainer.backgroundColor = KarhooUI.colors.white
+        view.addSubview(bottomContainer)
+        bottomContainer.heightAnchor.constraint(equalToConstant: 100).then { $0.priority = .defaultLow }.isActive = true
+        bottomContainer.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
```

Comment:
In dark mode the bottom of the screen has a dark line just underneath the container.
![IMG_2677855D085B-1](https://user-images.githubusercontent.com/86302636/225329162-49b63af5-eefb-4108-8356-240a81515508.jpeg)

Reply:
updated

Reply:
Why there is so big bottom padding? shouldn't be same as top instead of container height equal 100?

Reply:
The big bottom padding is due to safeAreaGuide usage. This is aligned to the design as well. Hard-coded container height is just for AutoLayout engine to set up properly. The value is ignored afterward, due to low priority.